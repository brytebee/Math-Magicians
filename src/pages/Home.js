const Home = () => (
  <div className="content-container">
    <div className="content">
      <h1>Home Page</h1>
      <p>This shall be our temporary Home page</p>
    </div>
  </div>
);

export default Home;
