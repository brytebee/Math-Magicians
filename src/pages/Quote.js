const Quote = () => (
  <div className="content-container">
    <div className="content">
      <h1>Quote Page</h1>
      <p>Our temporary Quote page</p>
    </div>
  </div>
);

export default Quote;
