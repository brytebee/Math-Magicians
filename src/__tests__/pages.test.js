import React from 'react';
import renderer from 'react-test-renderer';
import Home from "../pages/Home";
import Quote from "../pages/Quote";


describe('Test that all components are rendering just fine.', () => {

  it('Render the Home page.', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Render the Quote page', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})