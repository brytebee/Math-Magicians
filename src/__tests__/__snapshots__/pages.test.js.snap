// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test that all components are rendering just fine. Render the Home page. 1`] = `
<div
  className="content-container"
>
  <div
    className="content"
  >
    <h1>
      Home Page
    </h1>
    <p>
      This shall be our temporary Home page
    </p>
  </div>
</div>
`;

exports[`Test that all components are rendering just fine. Render the Quote page 1`] = `
<div
  className="content-container"
>
  <div
    className="content"
  >
    <h1>
      Quote Page
    </h1>
    <p>
      Our temporary Quote page
    </p>
  </div>
</div>
`;
